README.md

```diff
@@ -70,18 +70,18 @@ Current Rand versions are:
     changes and fixes.
 -   Version 0.8 was released in December 2020 with many small changes.
 
-A detailed [changelog](CHANGELOG.md) is available for releases.
+A detailed [changelog](CHANGELOG.md) is available for release.
```

Comment:
This isn't a grammar change: it's a change of meaning. Write "for each release" if you prefer.